tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/Responses/Response.cs

```diff
@@ -15,26 +15,35 @@ public class Response
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
     public List<string> ResponseErrors { get; set; }
 
+    [JsonPropertyName("next_steps")]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
+    public string? NextSteps { get; set; }
```

Comment:
Why do we need it to be plural?
It is just going to be a blob of string.

Reply:
Some of the more complex tool implementations do a lot of string building at various points. It seems easier for this to be a list rather than making the caller have to worry about string formatting when they append next steps multiple times?